kivy_garden/graph/__init__.py

```diff
@@ -572,7 +576,7 @@ def _redraw_x(self, *args):
         if self.xlabel:
             xlabel = self._xlabel
             if not xlabel:
-                xlabel = Label()
+                xlabel = Label(color = args[0])
```

Comment:
could you please explain why not using instead of `args[0]`, directly the Kivy property `self.x_label_color`?

Reply:
Actually, I think that works well too. I was only focusing on passing arguments to the respective Labels through the initial call to _redraw_all, but if we look at passing these arguments directly to the respective labels, then It would be better and more meaningful.